src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/AcceptanceDateMapper.java

```diff
@@ -0,0 +1,24 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import org.hl7.fhir.r4.model.Parameters;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.AcceptanceType;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.DateTimePeriod;
+import uk.nhs.digital.nhsconnect.nhais.model.fhir.ParameterNames;
+import uk.nhs.digital.nhsconnect.nhais.model.fhir.ParametersExtension;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+public class AcceptanceDateMapper implements FromFhirToEdifactMapper<DateTimePeriod> {
+
+    public DateTimePeriod map(Parameters parameters) {
+        return new DateTimePeriod(getAcceptanceDate(parameters), DateTimePeriod.TypeAndFormat.ACCEPTANCE_DATE);
+    }
+
+    private Instant getAcceptanceDate(Parameters parameters) {
+        String dateAsString = ParametersExtension.extractValue(parameters, ParameterNames.ACCEPTANCE_DATE);
+        return LocalDate.parse(dateAsString).atStartOfDay(ZoneOffset.UTC).toInstant();
```

Comment:
Shouldn't be UK time zone?

Reply:
From FHIR so ISO 8601 date. There are no time zones associated with dates.